chain/jsonrpc/Cargo.toml

```diff
@@ -41,3 +41,4 @@ dump_errors_schema = ["near-rpc-error-macro/dump_errors_schema"]
 adversarial = ["near-jsonrpc-primitives/adversarial"]
 nightly_protocol = ["near-primitives/nightly_protocol"]
 sandbox = []
+protocol_feature_routing_exchange_algorithm = []
```

Comment:
Why do we need this?

Reply:
There are two methods that can't be called unless you enable `protocol_feature_routing_exchange_algorithm`.